strawberryfields/gbs/plot.py

```diff
@@ -260,3 +266,79 @@ def plot_subgraph(subgraph: nx.Graph, size: int = 500) -> None:  # pragma: no co
     f = go.Figure(data=[g_edges, g_nodes], layout=layout)
 
     return f
+
+
+def plot_points(R: np.ndarray, sample: list, size: int = 500) -> None:  # pragma: no cover
+    """Creates a Plotly plot of the input points given their coordinates, highlighting an input
+    sample of points.
+
+    **Example usage:**
+
+    >>> R = np.random.normal(0, 1, (50, 50))
+    >>> sample = [1] * 10 + [0] * 40  # select first ten points
+    >>> plot_points(R, sample).show()
+
+    .. image:: ../../_static/normal_pp.png
+       :width: 40%
+       :align: center
+       :target: javascript:void(0);
+
+    Args:
+        R (np.array): Coordinate matrix. Rows of this array are the coordinates of the points.
+        sample (list[int]): subset of points to be highlighted, usually obtained from a sample
+
+    Returns:
+         Figure: Plotly figure of points with the sample highlighted
+    """
+    try:
+        import plotly.graph_objects as go
+    except ImportError:
+        raise ImportError(
+            "Plotly required for using plot(). Can be installed using pip install "
+            "plotly or visiting https://plot.ly/python/getting-started/#installation"
+        )
+    except RuntimeError:
+        print("Plotly unable to open display")
+        raise
+
+    layout = go.Layout(
+        showlegend=False,
+        hovermode="closest",
+        xaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
+        yaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
+        margin=dict(b=0, l=0, r=0, t=25),
+        height=size,
+        width=size,
+        plot_bgcolor="white",
+    )
+
+    points = go.Scatter(
+        x=R[:, 0],
+        y=R[:, 1],
+        mode="markers",
+        hoverinfo="text",
+        marker=dict(color="#F2F2F2", size=30, line=dict(color="black", width=1.5)),
+    )
+
+    points.text = [str(i) for i in range(len(R))]
+
+    s_x = []
+    s_y = []
+    sampled_points = [i for i in range(len(sample)) if sample[i] > 0]
+    for i in sampled_points:
```

Comment:
```suggestion
    for s in sample:
        if s > 0:
```
to save defining `sampled_points`?

Reply:
```suggestion
    for s in gbs.sample.modes_from_counts(sample):
```
I think this is kind of what we want from `modes_from_counts`. Also, this raises the question: why do we prefer samples to be input in the "list of counts" format as opposed to the "list of nodes" format (which equals the points selected)?

Reply:
The output of the samplers is in the "list of counts" format, so this should be the default input